File: lib/features/home/ui/category_icons.dart
import 'package:flutter/material.dart';

IconData categoryIcon(String categoryId) => switch (categoryId) {
      'electricity' => Icons.bolt,
      'water' => Icons.water_drop,
      'piped-gas' => Icons.local_fire_department,
      'lpg' => Icons.propane_tank,
      'mobile-postpaid' => Icons.smartphone,
      'mobile-prepaid' => Icons.sim_card,
      'dth' => Icons.satellite_alt,
      'broadband' => Icons.wifi,
      'landline' => Icons.phone,
      'fastag' => Icons.toll,
      'credit-card' => Icons.credit_card,
      'insurance' => Icons.health_and_safety,
      'loan-emi' => Icons.account_balance,
      'education' => Icons.school,
      'municipal-tax' => Icons.location_city,
      _ => Icons.receipt_long,
    };

File: src/features/home/ui/categoryIcons.ts
export const categoryIcon = (categoryId: string): string => {
  switch (categoryId) {
    case 'electricity': return 'flash';
    case 'water': return 'water';
    case 'piped-gas': return 'flame';
    case 'lpg': return 'cube';
    case 'mobile-postpaid': return 'phone-portrait';
    case 'mobile-prepaid': return 'card';
    case 'dth': return 'tv';
    case 'broadband': return 'wifi';
    case 'landline': return 'call';
    case 'fastag': return 'car';
    case 'credit-card': return 'card-outline';
    case 'insurance': return 'shield-checkmark';
    case 'loan-emi': return 'business';
    case 'education': return 'school';
    case 'municipal-tax': return 'location';
    default: return 'receipt';
  }
};
